src/components/Slideshow/Slideshow.styled.jsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+export const MainContainer = styled.div`
+    position: relative;
+    
+`;
+export const SlideshowContainer = styled.div`
```

Comment:
what does this name refers to? I'm trying to understand the "how"

Reply:
I named it "slideshow" (diapositivas), but to avoid misunderstandings, I modified it to slides-container.